crates/common/checkpoint_sync/src/lib.rs

```diff
@@ -47,29 +47,34 @@ pub async fn initialize_db_from_checkpoint(
     checkpoint_sync_url: Option<Url>,
 ) -> anyhow::Result<()> {
     if db.is_initialized() {
-        warn!("Starting Checkpoint Sync from existing DB. It is advised to start from a clean DB");
+        warn!("Starting checkpoint sync from existing DB. It is advised to start from a clean DB");
```

Comment:
```suggestion
        debug!("Checkpoint sync is already initialized");
```

This log doesn't make sense, as Checkpoint sync doesn't run if the database is already initialized

Reply:
We still need to sync from the existing db by querying peers to reach close to the head no? 

So if db already exists maybe we can say something accordingly?

In my mind, checkpoint sync is a two part process
1. Download initial state
2. Progress intial state to the head


@KolbyML what do you think

Reply:
I propose:

```suggestion
        warn!("DB is already initialized. Skipping checkpoint sync.");
```

Reply:
> We still need to sync from the existing db by querying peers to reach close to the head no?
> 
> So if db already exists maybe we can say something accordingly?
> 
> In my mind, checkpoint sync is a two part process
> 
> 1. Download initial state
> 2. Progress intial state to the head
> 
> @KolbyML what do you think

2. this isn't apart of checkpoint sync, this is just backfill sync from our latest state available. Checkpoint sync is just how we initialize a state if we have none

Reply:
> I propose this. I think it means exactly what this part of the code does.

This suggestion is good